db/migrate/20230403150639_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,6 @@
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
+    rename_column :likes, :user_id, :author_id
+  end
+end
```

Comment:
- [ ] Also here, Kindly write the query like this so the foreign key `author_id` will add relationships between user and like tables :


```ruby
  add_reference :likes, :author, null: false, foreign_key: { to_table: :users }
```